features/step_definitions/sriov.rb

```diff
@@ -120,3 +143,36 @@
     raise "Unable to delete the sriovnetwork" unless @result[:success]
   end
 end
+
+Given /^(SR-IOV resource injector|Admission webhook) is (enabled|disabled)$/ do | feature, status |
+  ensure_admin_tagged
+  project("openshift-sriov-network-operator")
+  if feature == 'SR-IOV resource injector'
+     testfeature = 'enableInjector'
+  else
+     testfeature = 'enableOperatorWebhook'
+  end
+  if status == 'enabled'
+     feature_status = true
+  else
+     feature_status = false
+  end
+  step %Q/I run the :patch admin command with:/, table(%{
+     | resource      | sriovoperatorconfigs.sriovnetwork.openshift.io |
+     | resource_name | default                                        |
+     | p             | {"spec":{"#{testfeature}": #{feature_status}}} |
+     | type          | merge                                          |
+   })
+  step %Q{the step should succeed}
+end
+
+Given /^I patch the sriov logs to "(.+?)"$/ do | loglevel |
```

Comment:
@zhaozhanqi I don't see this step use anywhere, in the future perhaps separating it into a different PR

Reply:
you can search
  `Given I patch the sriov logs to "0"`  in sriov.feature